src/constants/securedRoutes.js

```diff
@@ -14,6 +14,7 @@ import { getValueFromStorage } from '../utils';
 const Home = lazy(() => import('../pages/home'));
 const WatchList = lazy(() => import('../pages/watchList'));
 const SearchMovie = lazy(() => import('../pages/searchMovie'));
+const PlayerScreen = lazy(() => import('../pages/playerScreen'));
```

Comment:
Player screen can be deleted as we planned to open player in fullscreen mode only